src/Parser.js

```diff
@@ -789,7 +789,7 @@ Parser.prototype.parseSizeGroup = function(optional) {
     let res;
     if (!optional && this.nextToken.text !== "{") {
         res = this.parseRegexGroup(
-            /^[-+]? *(?:$|\d+|\d+\.\d*|\.\d*) *[a-z]{0,2}$/, "size");
+            /^[-+]? *(?:$|\d+|\d+\.\d*|\.\d*) *[a-z]{0,2}\s*$/, "size");
```

Comment:
There's no way to call `consumeSpaces()` here, this is before the tokens are created. It would need to be move into `this.parseRegexGroup`. And that would be a bigger project: ti would change the meaning. Please take commit as is.